vhdbuilder/packer/install-dependencies.sh

```diff
@@ -471,17 +471,23 @@ PATCHED_HYPERKUBE_IMAGES="
 1.15.10-hotfix.20200408.1
 1.15.11-hotfix.20200529.1
 1.15.11-hotfix.20200714.1
+1.15.11-hotfix.20200817.1
```

Comment:
idk how much additional space this'll take, but we shall see when build vhd

Reply:
==> azure-arm: + df -h
    azure-arm: Filesystem      Size  Used Avail Use% Mounted on
    azure-arm: udev            3.4G     0  3.4G   0% /dev
    azure-arm: tmpfs           695M  8.7M  687M   2% /run
    azure-arm: /dev/sda1        29G   27G  2.2G  93% /
    azure-arm: tmpfs           3.4G     0  3.4G   0% /dev/shm